### DISCUSSION TITLE
Calendar doesnt show up in Iphone / in Outlook it does!

### DISCUSSION THREAD

**Discussion by hevilp** [Kozea/Radicale#1451] [Q&A client]
Hi,

I got radicale 2.0.0 running on port 5232
I got nginx as reverse proxy running on port 800, no ssl.

With Outlook, I can connect on both ports without any problems.
On my Iphone, only the calendar on the 5232 port is shown. Both accounts are created successfully, but the nginx-account will not show up in the calendars.

Why?

**Comment by Unrud** [comment]
Can you provide the debug log output from Radicale? Preferable the log with and without Nginx.
